script.js

```diff
@@ -89,35 +120,35 @@ function validatePlayerSelection() {
 
 function playRound(playerSelection, computerSelection) {
   if (playerSelection === computerSelection) {
-    console.log(`\n%cResult: It's a tie!\n`, "font-size: 13px; font-weight:bold; color:orange");
+    console.log(`\n%cResult: A tie?! I idn\'t knew it was a formal event!\n`, "font-size: 13px; font-weight:bold; color:orange");
     return 0;
   } else if (
     (playerSelection === "rock" && computerSelection === "scissors") ||
     (playerSelection === "paper" && computerSelection === "rock") ||
     (playerSelection === "scissors" && computerSelection === "paper")
   ) {
-    console.log(`\n%cResult: You win :) ${selectUserWinsMessage()}\n`,
-    "font-size: 13px; font-weight:bold; color:green");
+
+    console.log(`\n%cResult: You win, ${selectUserWinsMessage()}\n`, "font-size: 13px; font-weight:bold; color:green");
     return 1;
   } else {
-    console.log(`\n%cResult: You lose :( ${selectComputerWinsMessage()}\n`,
-    "font-size: 13px; font-weight:bold; color:red");
+    console.log(`\n%cResult: You lose, ${selectComputerWinsMessage()}\n`, "font-size: 13px; font-weight:bold; color:red");
+
     return -1;
   }
 }
 
 function selectUserWinsMessage() {
   const userWinsMessages = [
-    "Great job", "Congratulation!", "You did it!", "Success!", "Victory!",
-    "Hooooray!", "Well done!", "You got it!"
+    "Lucky gess...", "Are you reading my code?", "You did it!How?!!!", "Victory!Somehow!",
+    "Congratulations!I don't understand how either...","Could you be more irritant?!"
   ];
   return userWinsMessages[Math.floor(Math.random() * userWinsMessages.length)];
 }
 
 function selectComputerWinsMessage() {
   const computerWinsMessages = [
-    "Your IA opponent wins.", "Keep calm and game on.", "Don't give up!",
-    "Maybe next time.", "Try harder.", "No worries.", "Maybe next time."
+    "Point for me!", "Are you even trying?", "Don't give up!", "Maybe next time.", "Try harder.",
+    "Am I the only one playng here?","Look at that! one more for me. Hahahaha","Luck is not on your side!Hahahaha"
```

Comment:
Spelling:
Am I the only one play**i**ng here?" - add "i"
"Look at that! **O**ne more for me. Hahaha" - big letter "0"
"Luck is not on your side!Hahahaha" - take out that "**hahaha**" (not needed) just leave: "**Luck is not on your side!**" (that is simple and to the pint)

Reply:
The AI is mocking the player. I believe that the laughter helps to make a poit.

Reply:
You're right! :) Let's keep it your way :)